### ISSUE TITLE
Duplicated rewards

### ISSUE THREAD

**Issue by Freudianity** [HabitRPG/habitica#543]
Each reward I add is duplicated in the reward view. This persists until I add a new reward, at which point the duplicate disappears, to be replaced by the new duplicate. So for example, I type CAKE:

cake
cake

I type STRAWBERRIES:

strawberries
strawberries
cake

It's not major, but it's irksome.

**Comment by Pandoro** [comment]
Have you tried refreshing after adding something? 

I know this is not a solution, but it works for many of the bugs there are currently.

**Comment by horusofoz** [comment]
What OS are you using? Windows XP/Vista7/8, Ubuntu, Mac OSX, Android, iOS, Blackberry, etc?

 What connection type are you using? Wired connection, Wifi, Wifi to mobile, 4/3/Next G?

What browser are you using? Internet Explorer 6/7/8/9/10, Chrome, Firefox, Opera?

**Comment by Freudianity** [comment]
Windows 7, Wifi, latest Chrome.

Refreshing does nothing. After further testing, deleting the top duplicated reward deletes the entire list.

**Comment by horusofoz** [comment]
Has this been ongoing for some time or just recent? 

Is it definitely Rewards specific or does similar behaviour occur when adding Habits/Dailys/Todos? 

Have you closed and reopened your web browser?

**Comment by Freudianity** [comment]
Definitely rewards specific. I only started yesterday, so I can't be specific with time. Web browser has been closed, computer has been rebooted.

**Comment by horusofoz** [comment]
Can you try clearing your browser cache, close, re-opren Chrome and see if
same behaviour repeats? Also what extensions do you have installed? I have
the same setup as yourself (Win7/Chome) without issue so this may be a
factor.

I haven't come across this issue previously so would be good to rule out
client side issues. These are the last probable client side factors I can
think of before we look to our side of the fence.
*
*
_Horusofoz_
*
*
"Live to learn. Learn to live." J.W.S.

**Comment by Pandoro** [comment]
@Freudianity Anything new?

You said after deleting the top one, the whole list was gone, I am wondering what happened after that? Did starting over fix this issue, or did you get duplicates again after starting over.

Furthermore, are you maybe logged in from several places?

**Comment by Pandoro** [comment]
I can confirm this as well now.

I deleted 1 reward (out of my three) and 2 were removed, the one I clicked and another one. So I removed the last one as I wanted to remove 2 rewards. Then I added the one I lost additionally. This was added twice and the old one also reappeared. I removed one of the two new ones and everything was gone again. 

Refreshing seems to fix stuff for me though. Meaning the duplicates are removed, but 1 actual reward remains.
